From 81db8a8e28ea1df24e3552e9c33a4656d62149ae Mon Sep 17 00:00:00 2001
From: Vince Chang <vince_chang@aspeedtech.com>
Date: Wed, 8 Jan 2025 09:58:28 +0800
Subject: [PATCH] obmc-ikvm: support ast2700 A1

For AST2700-A1:  (Temporary solution)
Node0 KVM and VM use 0x12060000
Node1 KVM and VM use 0x12062000

Signed-off-by: Vince Chang <vince_chang@aspeedtech.com>
---
 ikvm_input.hpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ikvm_input.hpp b/ikvm_input.hpp
index 0669f81..8bbe462 100644
--- a/ikvm_input.hpp
+++ b/ikvm_input.hpp
@@ -85,9 +85,9 @@ class Input
         "/sys/kernel/config/usb_gadget/obmc_hid1/UDC";
     /* @brief Path to the USB virtual hub */
     static constexpr const char* USB_VHUB_PATH =
-        "/sys/bus/platform/devices/12011000.usb-vhub";
+        "/sys/bus/platform/devices/12060000.usb-vhub";
     static constexpr const char* USB_VHUB_PATH_1 =
-        "/sys/bus/platform/devices/12021000.usb-vhub";
+        "/sys/bus/platform/devices/12062000.usb-vhub";
     /* @brief Retry limit for writing an HID report */
     static constexpr int HID_REPORT_RETRY_MAX = 5;
     /*
-- 
2.25.1

